examples/material-ui-pigment-css-nextjs-ts/next.config.mjs

```diff
@@ -8,6 +8,8 @@ export default withPigment(nextConfig, {
   theme: createTheme({
     cssVariables: true,
     colorSchemes: { light: true, dark: true },
+    defaultColorScheme: 'light',
+    colorSchemeSelector: 'class',
     typography: {
       fontFamily: 'var(--font-roboto)',
     },
```

Comment:
Base on Material UI docs, I think this should be:

```js
theme: createTheme({
    cssVariables: {
      colorSchemeSelector: 'class',
    },
    colorSchemes: { light: true, dark: true },
    typography: {
      fontFamily: 'var(--font-roboto)',
    },
```

Reply:
`defaultColorScheme: 'light',` is not required but I'm good if you want it to be explicit to the readers.